### ISSUE TITLE
It can converge?

### ISSUE THREAD

**Issue by Yeongtae** [Rayhane-mamah/Tacotron-2#162]
@Rayhane-mamah Wavenet model being trained using your latest parameter. 
[hparams.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2304666/hparams.zip)

![image](https://user-images.githubusercontent.com/13597229/44377745-28248100-a539-11e8-8c1f-8a23ae627529.png)

It seems to overfitting.

Could you share your tensorboard image?

In addition, every models which use 'raw', 'mulaw', 'mulaw-quantize' show the same pattern like above image.

**Comment by lkfo415579** [comment]
@Yeongtae Do you have any output wav from your WaveNet model? I trained a 150k WaveNet model but it produces noises waveform file. (Using default setting)

**Comment by Yeongtae** [comment]
@lkfo415579 
![image](https://user-images.githubusercontent.com/13597229/44380758-3037ed00-a548-11e8-9bca-302dff12c84f.png)
Here is it.

**Comment by Rayhane-mamah** [comment]
Hello @Yeongtae,

Those are 'mulaw-quantize' eval plots right?

As for eval loss, I can't really say it's useful at its current state really. I was hoping it would give some insight of whether or not the evaluation data is being modeled correctly but it's really not useful at all haha.. The thing is that evaluation loss is computed without any time constraint of timestep alignment, thus even if the model makes perfect audio prediction with few timesteps shift, the loss will be very big to the point we think the model has diverged.

while one could try giving first *x* as the true label and let the model do the rest of the predictions in an auto regressive manner, we prefer to use **test_inputs** during evaluation to evaluate overfitting. Is that making sense? 

Conclusion: current evaluation loss is not a valid metric for saying the model is diverging, for that one might use *test_inputs* parameter to actually feel real targets for comparison (to make teacher forced synthesis) and evaluate a one-to-one, time aligned loss value

**Comment by Yeongtae** [comment]
@Rayhane-mamah 

Those are 'raw' eval plots.

**Comment by Rayhane-mamah** [comment]
Alright that's good, feel free to share samples if you synthesize some :)

**Comment by Yeongtae** [comment]
[wavs.zip](https://github.com/Rayhane-mamah/Tacotron-2/files/2305521/wavs.zip)
Here,
There are eval samples in eval_dir.

**Comment by lkfo415579** [comment]
@Rayhane-mamah Thank you about the evaluation discussion. How long do you think a wavenet model is well trained? Would you provide your latest info about it? (tensorboard graph and synthesized waveform files?)

**Comment by Rayhane-mamah** [comment]
Hey @Yeongtae seems like your model has found the good minimum pretty fast, are you open to sharing the model? 60k steps in 24 hours seems slow for a V100..

@lkfo415579 I thought if would take more than 800k steps personally but @Yeongtae seems to want to prove me wrong :)

**Comment by Yeongtae** [comment]
@Rayhane-mamah I hope you were right.

**Comment by phuongbinhbk** [comment]
@Yeongtae could i ask you how many steps did you run for training tacotron, also what was the value of avg_loss at the checkpoints, such as 10k, 20k steps. I am doing with the tacotron model but it seems take too much time to converge

**Comment by phuongbinhbk** [comment]
@Yeongtae could i ask you how many steps did you run for training tacotron, also what was the value of avg_loss at the checkpoints, such as 10k, 20k steps. I am doing with the tacotron model but it seems take too much time to converge

**Comment by Yeongtae** [comment]
@phuongbinhbk I'm using 105k iteration check points for tacotron.
